src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/representation/RefusalRequestDTO.java

```diff
@@ -28,47 +22,7 @@
 
   @NotNull private UUID caseId;
 
-  @NotNull private Integer agentId;
-
-  @Size(max = 12)
-  @LoggingScope(scope = Scope.MASK)
-  private String title;
-
-  @Size(max = 60)
-  @LoggingScope(scope = Scope.MASK)
-  private String forename;
-
-  @Size(max = 60)
-  @LoggingScope(scope = Scope.MASK)
-  private String surname;
-
-  @Size(max = 60)
-  private String addressLine1;
-
-  @Size(max = 60)
-  @LoggingScope(scope = Scope.MASK)
-  private String addressLine2;
-
-  @Size(max = 60)
-  @LoggingScope(scope = Scope.MASK)
-  private String addressLine3;
-
-  @Size(max = 60)
-  @LoggingScope(scope = Scope.MASK)
-  private String townName;
-
-  @NotNull private Reason reason;
-
-  private Region region;
-
-  @Pattern(regexp = Constants.OPTIONAL_POSTCODE_RE)
-  private String postcode;
-
-  private UniquePropertyReferenceNumber uprn;
-
-  @NotNull private Boolean isHouseholder;
+  @NotNull private RefusalType reason;
```

Comment:
should we rename `reason` to `type` to fit with the backend terminology ?

Reply:
Not sure it's worth the knock-on hit to cc-ui